src/test/java/uk/gov/justice/hmpps/prison/repository/jpa/repository/BedAssignmentHistoriesRepositoryTest.java

```diff
@@ -71,6 +71,12 @@ public void findBadAssignmentHistory_forLocationAndDatePeriod() {
                         .assignmentDate(LocalDate.of(2020,4,3))
                         .assignmentDateTime(LocalDateTime.of(LocalDate.of(2020,4,3), LocalTime.of(11,0)))
                         .assignmentReason("ADM")
+                        .build(),
+                BedAssignmentHistory.builder()
+                        .livingUnitId(-16L)
+                        .assignmentDate(LocalDate.of(1985,4,3))
+                        .assignmentDateTime(LocalDateTime.of(LocalDate.of(1985,4,3), LocalTime.of(11,0)))
```

Comment:
don't change, I prefer just doing a `LocalDateTime.parse("2017-12-24T10:15:30")` or similar, find it much more readable, otherwise  I start wondering if the month / day is indexed from 0 or 1...